meteor/server/api/playout/actions.ts

```diff
@@ -165,6 +165,7 @@ export async function deactivateRundownPlaylistInner(cache: CacheForPlayout): Pr
 			previousPartInstanceId: null,
 			currentPartInstanceId: null,
 			holdState: RundownHoldState.NONE,
+			nextSegmentId: undefined,
```

Comment:
This should probably be in the $unset block below

Reply:
Fixed by https://github.com/tv2/tv-automation-server-core/commit/49ffd5df5f235b0269bc2d960da7721aab45e699